src/main/java/org/lsc/AbstractSynchronize.java

```diff
@@ -231,7 +231,7 @@ protected final boolean synchronize2Ldap(final Task task) {
 			LOGGER.info("If you want to avoid this message, " + "increase the time limit by using dedicated parameter.");
 		}
 
-		logStatus(counter);
+		logStatus(task.getName(), Task.Mode.clean.toString(), counter);
```

Comment:
There is an issue here that is fixed by:
```suggestion
		logStatus(task.getName(), Task.Mode.sync.toString(), counter);
```